### ISSUE TITLE
Callbacks of RectifiedAdam's learning rate

### ISSUE THREAD

**Issue by triper1022** [tensorflow/addons#2542]
**Describe the feature and the current behavior/state.**
I am trying use your sample code to track the RectifiedAdam's learning rate.
But I used the default setting and I can't get the results.
Would you tell how to get it work? Thanks a lot.

```
# Load example MNIST data and pre-process it
(x_train, y_train), (x_test, y_test) = tf.keras.datasets.mnist.load_data()
x_train = x_train.reshape(-1, 784).astype("float32") / 255.0
x_test = x_test.reshape(-1, 784).astype("float32") / 255.0

# Limit the data to 1000 samples
x_train = x_train[:1000]
y_train = y_train[:1000]
x_test = x_test[:1000]
y_test = y_test[:1000]

batch_size=128
steps_per_epoch = 1000//batch_size

!pip install tensorflow-addons
import tensorflow_addons as tfa

class CustomCallback(keras.callbacks.Callback):
    def __init__(self):
        self.batch = 0
    def on_train_batch_end(self, batch, logs=None):
        lr = float(tf.keras.backend.get_value(self.model.optimizer.learning_rate))
        print("For batch {:3}, lr is {:7.2f}.".format(self.batch, lr))
        self.batch += 1

    
radam = tfa.optimizers.RectifiedAdam(
    learning_rate = 1e-3,
    weight_decay=0.01,
    total_steps=steps_per_epoch,
    warmup_proportion=0.1,
    min_lr=1e-5)

# Define the Keras model to add callbacks to
def get_model():
    model = keras.Sequential()
    model.add(keras.layers.Dense(1, input_dim=784))
    model.compile(
        optimizer=radam,
        loss="mean_squared_error",
        metrics=["mean_absolute_error"],
    )
    return model

model = get_model()
history = model.fit(
    x_train,
    y_train,
    batch_size=128,
    epochs=2,
    verbose=0,
    validation_split=0.1,
    callbacks=[CustomCallback()],
)

# For batch   0, lr is    0.00.
# For batch   1, lr is    0.00.
# For batch   2, lr is    0.00.
# For batch   3, lr is    0.00.
# For batch   4, lr is    0.00.
# For batch   5, lr is    0.00.
# For batch   6, lr is    0.00.
# For batch   7, lr is    0.00.
# For batch   8, lr is    0.00.
# For batch   9, lr is    0.00.
# For batch  10, lr is    0.00.
# For batch  11, lr is    0.00.
# For batch  12, lr is    0.00.
# For batch  13, lr is    0.00.
# For batch  14, lr is    0.00.
# For batch  15, lr is    0.00.
```
